commit eb04b4977d16c31435449a9796189f341e7dd7d3
Author: Valentina Mukhamedzhanova <vmukhame@redhat.com>
Date:   Thu Jul 10 12:05:04 2014 +0200

    yum manpage: move 'history info' description to its proper place.

diff --git a/docs/yum.8 b/docs/yum.8
index 86df6b0..998a5ad 100644
--- a/docs/yum.8
+++ b/docs/yum.8
@@ -535,6 +535,13 @@ The info/list/summary commands take either a transaction id or a package (with
 wildcards, as in \fBSpecifying package names\fP), all three can also be passed
 no arguments. list can be passed the keyword "all" to list all the transactions.
 
+The info command can also take ranges of transaction ids, of the form start..end,
+which will then display a merged history as if all the transactions in the range
+had happened at once\&.
+.br
+Eg. "history info 1..4" will merge the first four transactions and display them
+as a single transaction.
+
 The packages-list/packages-info commands takes a package  (with wildcards, as in
 \fBSpecifying package names\fP). And show data from the point of view of that
 package.
@@ -779,12 +786,6 @@ Checks the local rpmdb and produces information on any problems it finds. You
 can pass the check command the arguments "dependencies", "duplicates", "obsoletes" or "provides",
 to limit the checking that is performed (the default is "all" which does all).
 
-The info command can also take ranges of transaction ids, of the form
-start..end, which will then display a merged history as if all the
-transactions in the range had happened at once\&.
-.br
-Eg. "history info 1..4" will merge the first four transactions and display them
-as a single transaction.
 .IP
 .IP "\fBhelp\fP"
 Produces help, either for all commands or if given a command name then the help
